package-lock.json

```diff
@@ -80371,14 +80365,10 @@
         "js-tokens": "^3.0.0 || ^4.0.0"
       }
     },
-    "lottie-ios": {
-      "version": "3.5.0",
-      "peer": true
-    },
     "lottie-react-native": {
-      "version": "5.1.6",
-      "resolved": "https://registry.npmjs.org/lottie-react-native/-/lottie-react-native-5.1.6.tgz",
-      "integrity": "sha512-vhdeZstXMfuVKwnddYWjJgQ/1whGL58IJEJu/iSf0XQ5gAb4pp/+vy91mdYQLezlb8Aw4Vu3fKnqErJL2hwchg==",
+      "version": "6.3.1",
+      "resolved": "https://registry.npmjs.org/lottie-react-native/-/lottie-react-native-6.3.1.tgz",
+      "integrity": "sha512-M18nAVYeGMF//bhL27D2zuMcrFPH0jbD/deBvcWi0CCcfZf6LQfx45xt+cuDqwr5nh6dMm+ta8KfZJmkbNhtlg==",
       "requires": {
```

Comment:
Not sure why this file I generated is different from this branch.

Replace package-lock.json with this done:

[package-lock.json.zip](https://github.com/Expensify/App/files/12840510/package-lock.json.zip)